Examples/UIExplorer/js/XHRExampleFetch.js

```diff
@@ -63,6 +81,13 @@ class XHRExampleFetch extends React.Component {
       </View>
     ) : null;
 
+    var responseHeaders = this.responseHeaders ? (
+      <View style={{marginTop: 10}}>
+        <Text style={styles.label}>Server response Headers:</Text>
```

Comment:
nit: headers probably doesn't need to be capitalized?

Either:
Server response headers
or:
Server Response Headers